.github/workflows/test_release.yaml

```diff
@@ -2,6 +2,28 @@
 name: Test / Release
 on: [push, pull_request]
 jobs:
+  dockerbuild:
```

Comment:
This lets us build the docker image once and reference it in each kind version rather than rebuilding the docker image for every kind version.